Sources/Gtk/Widgets/Orientable.swift

```diff
@@ -8,10 +8,10 @@ public protocol Orientable: AnyObject {
 extension Orientable {
     public var orientation: Orientation {
         get {
-            gtk_orientable_get_orientation(opaquePointer).toOrientation()
+            return Orientation(from: gtk_orientable_get_orientation(opaquePointer))
         }
         set {
-            gtk_orientable_set_orientation(opaquePointer, newValue.toGtkOrientation())
+            gtk_orientable_set_orientation(opaquePointer, newValue.toGtk())
         }
     }
 }
```

Comment:
I think you can do `extension Orientable where Self: GObjectRepresentable` here and then you could use the property wrapper

Reply:
Yeah true, missed this one (I only made those changes in the pass where I was fixing compile errors, and didn't remember it when doing a pass to replace manual enum properties with the property wrapper)

Reply:
Sad, it counts as a stored property :(

Reply:
ah yeah true :facepalm: